crawler.go

```diff
@@ -300,6 +316,20 @@ func (c *Crawler) requestTimeout() time.Duration {
 	return 30 * time.Second
 }
 
+func (c *Crawler) secondsBetweenRetries() int {
+	if v := c.SecondsBetweenRetries; v > 0 {
+		return v
+	}
+	return 10
+}
+
+func (c *Crawler) maxRetries() int {
```

Comment:
Why do we need this accessor? It seems that if this was:

```go
func (c *Crawler) maxRetries() int { return c.MaxRetries }
```

would work too, and at that point, might as well remove the method?